controllers/artists.controller.js

```diff
@@ -75,7 +75,24 @@ const getProfile = async (req, res) => {
 };
 
 const updateArtist = async (req, res) => {
-  res.json('Update artist profile');
+  const { _id: artistId } = req.user;
+  const { name, email, password, genre, bio } = req.body;
+  const artist = await Artist.findOne({ _id: artistId });
+  if (!artist) {
+    return res.status(HTTP_STATUS.NOT_FOUND).json({ error: 'Artist cannot be found. Try again' });
+  }
+  const updatedArtist = await Artist.findOneAndUpdate(
```

Comment:
Use just update. There is no need to find since you already found the artist.

```suggestion
  const updatedArtist = await Artist.update(
```

Reply:
done on 39e6e83